### PR TITLE
EPMRPP-87634 || Add <meta> tag with value "noindex" to "Contact us" pages

### PR DESCRIPTION
**Pull request by Vadim73i**
### PR Checklist

* [ ] Have you followed the guidelines in our [Dev Guide](../#readme)
* [ ] Have you lint your code locally prior to submission (`npm run lint`)?
* [ ] Have you run prettier (`npm run format`)?
* [ ] Have you checked that app can be built (`npm run build`)?
* [ ] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

### CODE REVIEW

File: src/components/Layout/Seo/Seo.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ interface SeoProps {
   title?: string;
   image?: string;
   description?: string;
+  noindex?: boolean;
```
Comment (AmsterGet) [inline src/components/Layout/Seo/Seo.tsx]: Let's keep using camelCase for props and variables.

File: src/templates/contact-us/contact-us.tsx [hunk 2]
```diff
@@ -13,10 +16,15 @@ interface ContextProps {
   };
 }
 
-const ContactUs: FC<PageProps<null, ContextProps>> = ({ pageContext: { config } }) => (
-  <Layout seoData={SEO_DATA.contactUsGeneral} className="features-page-layout">
-    <ContactUsPage config={config} />
-  </Layout>
-);
+const ContactUs: FC<PageProps<null, ContextProps>> = ({ pageContext: { config } }) => {
+  const location = useLocation();
```
Comment (AmsterGet) [inline src/templates/contact-us/contact-us.tsx]: You can use property `url` from the `config` and remove this hook usage.
I see that for some reason this property doesn't mentioned in the props, so please add it to the interface.
It is propagated via `createContactUsConfig` from `src/utils/contactUsConfig.ts`.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Comment by ViktorSoroka07** [comment]
Run prettier command for `npm run format:prettier`. Set it up for your editor

File: src/templates/contact-us/contact-us.tsx [hunk 3]
```diff
@@ -2,21 +2,31 @@ import React, { FC } from 'react';
...
-  <Layout seoData={SEO_DATA.contactUsGeneral} className="features-page-layout">
-    <ContactUsPage config={config} />
-  </Layout>
-);
+const ContactUs: FC<PageProps<null, ContextProps>> = ({ pageContext: { config } }) => {
+  const noIndex = config?.url !== CONTACT_US_GENERAL_PATH;
+
+  return (
+    <Layout
+      seoData={{ noIndex: noIndex, ...SEO_DATA.contactUsGeneral }}
```
Comment (AmsterGet) [inline src/templates/contact-us/contact-us.tsx]:
```suggestion
      seoData={{ noIndex, ...SEO_DATA.contactUsGeneral }}
```

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**